### PR TITLE
Add doc and test for activation function

### PR DESCRIPTION
**Pull request by bagustris**

### CODE REVIEW

File: tests/test_activation_functions.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import torch
+import torch.nn as nn
+
+# Test activation functions
+activation_functions = {
+    'relu': nn.ReLU(),
+    'leaky_relu': nn.LeakyReLU(),
+    'tanh': nn.Tanh(),
+    'sigmoid': nn.Sigmoid(),
+    'elu': nn.ELU(),
```
Comment (felixbur) [inline tests/test_activation_functions.py]: @bagustris , i think here are more activation functions than i originally added to nkululeko (because they deemed not very important to me). So i guess either, we stick here to relu, leaky_relu, tanh and sigmoid, or you add the other functions to the mlp and mlp_reg code (which is fine by me obvisously)
